tools/device_file_generator/avr_io.py

```diff
@@ -1281,12 +1281,12 @@
 		[
 			{ "port": "B", "id": "2", "int": "0" },
 		],
-		"spi":
-		[
-			{ "port": "B", "id": "1", "name": "miso", "dir": "in" },
-			{ "port": "B", "id": "2", "name": "mosi", "dir": "out" },
-			{ "port": "B", "id": "3", "name": "sck", "dir": "out" }
-		],
+		#"spi":
+		#[
+		#	{ "port": "B", "id": "1", "name": "miso", "dir": "in" },
+		#	{ "port": "B", "id": "2", "name": "mosi", "dir": "out" },
+		#	{ "port": "B", "id": "3", "name": "sck", "dir": "out" }
+		#],
 		"usi":
 		[
 			{ "port": "B", "id": "2", "name": "usck", "dir": "out" },
```

Comment:
Those have a slight problem:
Atmel calls that SPI and TWI although it is only implemented through the USI. This confuses the GPIO driver which assumes that there is SPI driver which isn't available. Without an USI driver we cannot use those pins.